src/utils/getLocationsApiData.js

```
@@ -0,0 +1,39 @@
+    if (result.error === false && Array.isArray(result.data)) {
+      return result.data.map((country) => country.name)
+    } else {
+      return []
+    }
+  } catch (error) {
+    throw createError(500, INTERNAL_SERVER_ERROR)
+  }
+}
+
+const getCitiesDataByCountry = async (countryName) => {
+  try {
```

ESLint

[error] Expected linebreaks to be 'CRLF' but found 'LF'.

(linebreak-style)